install.sh

```diff
@@ -3,6 +3,8 @@
 
 lscolors_data_dir="${XDG_DATA_HOME:-$HOME/.local/share}"
 
+[[ -d $lscolors_data_dir ]] || mkdir -p $lscolors_data_dir > /dev/null 2>&1
```

Comment:
```suggestion
[[ -d $lscolors_data_dir ]] || mkdir --verbose --parents $lscolors_data_dir > /dev/null 2>&1
```
This is a good idea. I'd like to notify users that we are doing this, so I suggested adding the `--verbose` switch.

Reply:
Sure, will do.